engine.py

```diff
@@ -161,9 +172,12 @@ def _set_project(self):
         self.log_info("Setting Softimage project to '%s'" % proj_path)
 
         try:
-            # test to see if the project has already been set to this path:
+            # test to see if the project has already been set to this path
+            # Application.ActiveProject.Path returns a unicode object. If the path contains
+            # non-ascii characters, the comparison will fail since the str and unicode objects
+            # cannot be compared, so we convert the unicode object to a utf-8 string.
             if (Application.ActiveProject.Path 
-                and os.path.normpath(Application.ActiveProject.Path).lower() == os.path.normpath(proj_path).lower()):
+                and os.path.normpath(Application.ActiveProject.Path).lower().encode("utf-8") == os.path.normpath(proj_path).lower()):
```

Comment:
`Application.ActiveProject.Path` seems like it always returns a unicode object. Python tries to compare it with the string object when possible, but it it contains non-ascii characters that it cannot compare with a utf-8 encoded string object, then it returns "not equal" by default since they cannot be compared. So not sure there's an additional pre-check we can do here that's worthwhile.